heron/stmgr/tests/cpp/server/stmgr_unittest.cpp

```diff
@@ -228,7 +228,7 @@ void StartDummyStMgr(EventLoopImpl*& ss, DummyStMgr*& mgr, std::thread*& stmgr_t
 
   mgr = new DummyStMgr(ss, options, stmgr_id, LOCALHOST, stmgr_port, LOCALHOST, tmaster_port,
                        shell_port, instances);
-  mgr->Start();
+  EXPECT_EQ(0, mgr->Start()) << stmgr_port;
```

Comment:
When the check fails, it prints stmgr_port.
The ci seems always failing after adding the checking. I doubt if this test ever run correctly.